### DISCUSSION TITLE
9월12일 회의

### DISCUSSION THREAD

**Discussion by sangwon-1** [receiptAutoScan/receipt-backend#16] [회의안]
결정 사항

1. OCR 사용 방향성 결정하기 ( 템플릿 추가, 코드를 통한 사용)
2. 언어 모델 어떤 것을 사용할지 
3. 포스기를 통한 자료 일별로 조회가 가능한지

**Comment by JS-HAN214** [comment]
확인했습니다!

**Comment by ej20001000** [comment]
포스기 엑셀 export 시 건 별로 export가 가능한지 궁금합니다! 서버반 및 기획반 강사님들께서 매출 정보를 받는게 법적 문제가 없다 하여 가능하면 자세한 정보들을 저장하여 매출, 지출, 순수익 뿐만 아니라 추후 품목 별 통계, 일별 / 월별 통계 등 고도화를 할 수 있는 방향이 나을 것 같습니다.
